.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+Contributing to Conan Center Index
```

Comment:
The pipeline is not prepared to process this file. Why should it be under the .github folder?

Reply:
This is a file with a special meaning. GitHub will provide a link to this explanation for contributors

Reply:
yeah, now I remember. Should it be under *.github* or in the root folder? Maybe we can put it under *docs* as it is considered documentation as well https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

Reply:
done!